src/main/java/com/codeday/productivity/entity/Goal.java

```diff
@@ -0,0 +1,55 @@
+package com.codeday.productivity.entity;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.Column;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.Instant;
+
+@Entity
+@Table(name = "GOAL_TBL")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Goal {
+
+    @Id
+    @GeneratedValue
+    private int id;
+
+    private String title;
+
+    private String description;
+
+    @Column(columnDefinition = "TIMESTAMP")
+    private Instant startDate;
+
+    @Column(columnDefinition = "TIMESTAMP")
+    private Instant endDate;
```

Comment:
Is this nullable?  We might want to keep it nullable, as goals could potentially not have an end data.

Reply:
It is nullable by default since I have not annotated it to be non-nullable. I thought goal would have an end date.